File: BE/API/Models/DetalleCobranza.cs
namespace API.Models;

public class DetalleCobranza
{
    public Guid Id { get; set; }
    public double ImporteCobrado { get; set; }
    public double ImporteRetenido { get; set; }
    public double ImporteTotal { get; set; }
    public Guid FacturaId { get; set; }
    public DateTime FechaPago { get; set; }
    public Guid TipoPagoId { get; set; }
    public Guid CobranzaId { get; set; }
}

File: FE/src/interfaces/detalleCobranza.ts
import { Cobranza } from "./cobranza";
import { Factura } from "./factura";
import { TipoPago } from "./tipoPago";

export interface DetalleCobranza {
    id: string,
    importeCobrado: number,
    importeRetenido: number,
    importeTotal: number,
    //Factura: Factura,
    facturaId: string,
    fechaPago: Date,
    //TipoPago: TipoPago,
    tipoPagoId: string,
    cobranzaId: string
}
